www/src/utils/typography.js

```diff
@@ -60,17 +73,16 @@ const options = {
         paddingBottom: `0.1em`,
       },
       ".gatsby-highlight": {
-        // background: `hsla(23, 60%, 97%, 1)`,
         background: colors.a[0],
-        border: `1px solid ${colors.a[1]}`,
+        boxShadow: `inset 0 0 0 1px ${colors.a[1]}`,
         borderRadius: `${presets.radius}px`,
-        padding: rhythm(options.blockMarginBottom),
```

Comment:
I was playing around with pulling the highlighted code blocks into the containers horizontal padding and wanted things to line up. Not too strongly attached to this either btw :D

![image](https://user-images.githubusercontent.com/21834/29216358-a9a2feb2-7eae-11e7-8ec6-5938c779188a.png)